pkg/runtime/runtime.go

```diff
@@ -302,21 +288,55 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 	a.populateSecretsConfiguration()
 	// Create and start internal and external gRPC servers
 	grpcAPI := a.getGRPCAPI()
+
 	err = a.startGRPCAPIServer(grpcAPI, a.runtimeConfig.APIGRPCPort)
 	if err != nil {
 		log.Fatalf("failed to start API gRPC server: %s", err)
 	}
 	log.Infof("API gRPC server is running on port %v", a.runtimeConfig.APIGRPCPort)
+	a.grpcExternalAPI = grpcAPI
 
+	// Start HTTP Server
+	a.startHTTPServer(a.runtimeConfig.HTTPPort, a.runtimeConfig.ProfilePort, a.runtimeConfig.AllowedOrigins, pipeline)
+	log.Infof("http server is running on port %v", a.runtimeConfig.HTTPPort)
+
+	grpcAPI = a.getGRPCAPI()
```

Comment:
please do not set new gRPCAPI. use one for now. spliting grpc api server object is the separate issue.